app/src/main/java/com/codesoom/assignment/DemoHttpHandler.java

```diff
@@ -0,0 +1,169 @@
+package com.codesoom.assignment;
+
+import com.codesoom.assignment.models.Task;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+
+import java.io.*;
+import java.net.URI;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class DemoHttpHandler implements HttpHandler {
+
+    private List<Task> tasks = new ArrayList<>();
+
+    ObjectMapper mapper = new ObjectMapper();
+
+    @Override
+    public void handle(HttpExchange exchange) throws IOException {
+
+        String method = exchange.getRequestMethod();
+        URI requestURI = exchange.getRequestURI();
+        String path = requestURI.getPath();
+        System.out.println(method + " " + path);
+
+        InputStream inputStream = exchange.getRequestBody();
+        String body = new BufferedReader(new InputStreamReader(inputStream))
+                .lines()
+                .collect(Collectors.joining("\n"));
+
+        // path가 아이디를 가지는지 여부
+        boolean hasTaskId = Pattern.matches("/tasks/[0-9]+$", path);
+
+        // 콘텐츠 초기화
+        String content = null;
+
+        // GET /tasks
+        if (method.equals("GET") && path.equals("/tasks")) {
```

Comment:
이렇게 바꾸면 어떤 장점이 있을까요? 어떤 문제를 예방하게 될까요?
```suggestion
        if ("GET".equals(method) && path.equals("/tasks")) {
```

Reply:
코드에서 method라는 변수가 request에서 가져오기 때문에 null 값을 가질 수 있을 것 같습니다. method가 null 값을 가지면 문자열 비교를 못하고 에러를 발생시킬 것 같습니다. `GET` 을 왼쪽이 위치시켜서 에러를 방지할 수 있을 것 같습니다.